### PR TITLE
Calculator component

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this PR I did the following👇🏻: 

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new `Calculator.js` file.
- In `Calculator.js` , create a React component that matches this design.
- Develop the UI of the calculator.
- Use class-based components.
- Import `Calculator.js` and use it in the main component (App.js).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (tufoinnkuo10) [inline src/App.js]: - [ ] kindly ensure that your components are class based. 
example
```
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @shahira-sadat 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well followed

Required Changes ♻️
Check the comments under the review

tufoin


Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.js (hunk 1 above)
Comment (shahira-sadat) [reply src/App.js]: Done ✅

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,17 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
```
Comment (mateo951) [inline src/App.js:4]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @shahira-sadat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🤓 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**